controllers/news.controllers.js

```diff
@@ -0,0 +1,65 @@
+const { findingTopics, findingArticleId, findingUsers, patchingArticleId, findingArticles, findingComments } = require('../models/news.models.js')
+
+exports.getTopics = (req, res, next) => {
+    findingTopics().then((arrayOfTopics) => {
+        res.status(200).send({ topics: arrayOfTopics })
+    })
+        .catch((err) => {
+            next(err);
+        });
+}
+
+exports.getArticles = (req, res, next) => {
+    const { topic } = req.query;
+
+    findingArticles(topic).then((result) => {
+        res.status(200).send({ result })
+    })
```

Comment:
make sure you are sending data back on a key that describes the data

"`result`" could be anything! "`articles`" would be better here!